atomic_reactor/inner.py

```diff
@@ -434,7 +437,11 @@ def __init__(
         :param plugin_files: load plugins also from these files
         :param client_version: osbs-client version used to render build json
         """
+        self.build_dir = build_dir
```

Comment:
Could this be `self._build_dir` to avoid accidents with "build_dir" vs "build_dirs"?

Reply:
Fixed.